.github/workflows/scripts/publish_docs.sh

```diff
@@ -2,19 +2,29 @@
 
 set -euv
 
-openssl aes-256-cbc -K $encrypted_None_key -iv $encrypted_None_iv -in .ci/assets/pulp-infra.enc -out ~/.ssh/pulp-infra -d
+export WORKSPACE=$GITHUB_WORKSPACE
+export BRANCH=$GITHUB_REF_NAME_SLUG
+
+mkdir ~/.ssh
+echo "$PULP_DOCS_KEY" > ~/.ssh/pulp-infra
 sudo chmod 600 ~/.ssh/pulp-infra
```

Comment:
this should not need sudo.
Also can we use umask to create files and directories with the proper permissions to begin with?

Reply:
I agree but I'm merely trying to get this script running on Github Actions in this PR. How about if I file an issue to fix up this script?

Reply:
I'll remove sudo here though.

Reply:
Let's revisit that when we talk about running it on a developers local machine. I don't really like it to mess around with my .ssh directory at all. However i found no equivalent of GPG_HOME for ssh.